cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/metrics.rst

```diff
@@ -98,114 +373,162 @@ The general form of a metrics request is::
 
    * - Parameter
      - Description
-   * - ``<scope>``
-     - Either ``system`` (system metrics) or ``user`` (user-defined metrics)
    * - ``<context>``
-     - Hierarchy of context; see `Available Contexts`_
-   * - ``<run-id>``
-     - Run-ID of the program; see `Querying by Run-ID`_
+     - Metrics context to search within
+   * - ``<tags>`` (Optional)
+     - Comma-separated :ref:`tag list <http-restful-api-metrics-tag-list>` by which to group 
+       results (optional)
    * - ``<metric>``
-     - Metric being queried; see `Available Metrics`_
+     - Metric being queried
    * - ``<time-range>``
-     - A `Time Range`_ or ``aggregate=true`` for all since the Application was deployed
+     - A :ref:`time range <http-restful-api-metrics-time-range>` or ``aggregate=true`` for 
+       all since the Application was deployed
 
 .. rubric:: Examples
+
 .. list-table::
    :widths: 20 80
    :stub-columns: 1
 
    * - HTTP Method
-     - ``GET <base-url>/metrics/system/apps/HelloWorld/flows/``
-       ``WhoFlow/flowlets/saver/process.busyness?aggregate=true``
+     - ``GET <base-url>/metrics/query?context=namespace.default.app.HelloWorld.flow.``
+       ``WhoFlow.flowlet.saver&metric=system.process.busyness?aggregate=true``
    * - Description
-     - Using a *System* metric, *process.busyness*
+     - Using a *System* metric, *system.process.busyness*
    * - 
      - 
    * - HTTP Method
-     - ``GET <base-url>/metrics/user/apps/HelloWorld/flows/``
-       ``WhoFlow/runs/13ac3a50-a435-49c8-a752-83b3c1e1b9a8/flowlets/saver/names.bytes?aggregate=true``
+     - ``GET <base-url>/metrics/query?context=namespace.default.app.HelloWorld.flow.``
+       ``WhoFlow.runs.13ac3a50-a435-49c8-a752-83b3c1e1b9a8.flowlet.saver&metric=user.names.bytes?aggregate=true``
    * - Description
-     - Querying the *User-Defined* metric *names.bytes*, of the Flow *saver*, by its run-ID
+     - Querying the *User-defined* metric *names.bytes*, of the Flow *saver*, by its run-ID
    * - 
      - 
    * - HTTP Method
-     - ``GET <base-url>/metrics/user/apps/HelloWorld/services/``
-       ``WhoService/runnables/WhoRun/names.bytes?aggregate=true``
+     - ``GET <base-url>/metrics/query?context=namespace.default.app.HelloWorld.services``
+       ``WhoService.runnables.WhoRun&metric=user.names.bytes?aggregate=true``
    * - Description
-     - Using a *User-Defined* metric, *names.bytes* in a Service's Handler
+     - Using a *User-defined* metric, *names.bytes* in a Service's Handler
 
-.. rubric:: Comments
-
-The scope must be either ``system`` for system metrics or ``user`` for user-defined metrics.
-
-System metrics are either Application metrics (about Applications and their Flows, Procedures, MapReduce, Workflows and Workers) or they are Data metrics (relating to Streams or Datasets).
 
-User metrics are always in the Application context.
+.. rubric:: Comments
```

Comment:
No, my intention was that they were all on the same level...and the comments was just commentary on method. But I'll look and see how we can get rid of it ("Comments"). They are rubrics so they don't add another level to the table of contents.